debian/changelog

```diff
@@ -1,3 +1,9 @@
+django-testscenarios (0.10-1~bpo9+2) UNRELEASED; urgency=medium
```

Comment:
It is not possible to upload directly to stretch-backports, which is what this changelog version implies. All uploads must go through unstable first which means a version of 0.10-2 - when that migrates to testing, 0.10-2~bpo9+1 would be uploaded to stretch-backports.
Please remove the change to debian/changelog from this pull request as this will cause the least merge conflicts / re-editing when the upload is actually prepared.